lang/pl.json

```diff
@@ -1099,7 +1100,9 @@
   "Fields.name": "Nazwa",
   "Fields.name.description": "Nazwy są wyświetlane publicznie i używane wszędzie tam, gdzie ten profil pojawia się publicznie, jak np. wpłaty, komentarze do aktualizacji, publiczne informacje o wydatkach itp.",
   "Fields.website": "Strona",
-  "File.NoFilename": "Brak nazwy pliku",
```

Comment:
Looks like we're resting the translation for this string. The right strategy when rebasing translations on master is to keep `main`'s change (`-Xtheirs`) then rebuild

Reply:
For what it's worth, that's what I tried to do and must've simply overlooked this change -- thanks for catching it!